### ISSUE TITLE
Quake - No Audio - GLQuake crashes at launch (2310)

### ISSUE THREAD

**Issue by cjfcollins** [ValveSoftware/Proton#232]
Intel® Pentium(R) CPU G3240 @ 3.10GHz × 2 
7.7 GiB RAM
GeForce GTX 750 Ti/PCIe/SSE2 nvidia-driver-396.51
Gnome 3.28.2

If I attempt to launch GLQuake, it crashes immediately. 
Quake (winquake) has no audio

The game also does not fill the entire screen. ![(see screenshot)](https://user-images.githubusercontent.com/13816374/44508684-096ed900-a664-11e8-930c-12a10fcd1395.png) 
This can be fixed changing my native resolution to match the game resolution.

**Comment by rombert** [comment]
GLQuake also crashes at startup for me

Using Steam on openSUSE Tumbleweed, Kernel 4.18, Intel GPU (i7-4810MQ).
Monitor: W541 laptop display ( 15.6 in, 1920x1080 )
Display driver: i915
```
$ rpm -q llvm Mesa
llvm-6.0.1-1.1.x86_64
Mesa-18.1.5-204.2.x86_64
```

**Comment by aeikum** [comment]
GLQuake is also broken on Windows, don't use that build. https://steamcommunity.com/app/2310/discussions/0/359543951733236839/ https://steamcommunity.com/app/2310/discussions/0/1471967615882970708/

**Comment by whizse** [comment]
Probably needs the MESA_EXTENSION_MAX_YEAR override, and whatever the nvidia equivalent is.

**Comment by P4NCH1** [comment]
Was retesting some games and I notice that this one [doesn't work anymore in SteamOS](https://www.youtube.com/watch?v=aiKjh_u25_0).

## System Information
- GPU: AMD Radeon R7 Kaveri 1GB
- Driver/LLVM version: Mesa 17.2.4/5.0.0
- Kernel version:  4.14.0-0.steamos.81-amd64
- [Full system information report](https://gist.github.com/P4NCH1/93502637c9e1b0fceda250dd5dfeb52b#file-steamos-system-information)
- Proton version: 3.7-3 and 3.7-6 Beta

## Symptoms
Game doesn't even start no matter what option you choose (Quake, QuakeWorld, GLQuake, GLQuakeWorld). Tested with Proton versions 3.7-3 and 3.7-6 Beta.

## Reproduction
1. Start game
2. Choose lauch option
3. Nothing hapens
3. Steam client says "synchronizing"

## NOTE:
On this same PC, with Steam client v1535062307 was working perfect.
On a notebook with MX Linux 17.1 64bits with Steam client v1534979715 (when I tested it) was working perfect too.

**Comment by aeikum** [comment]
@P4NCH1 Do Proton games work in general on that system? It looks like the Wine prefix got messed up somehow. Uninstalling and re-installing Quake should reconfigure the prefix for you. (You can do this manually by deleting the compatdata folder for the game at `~/.steam/steam/steamapps/compatdata/2310`.)

**Comment by P4NCH1** [comment]
Yes, I still didn't test again all that [works before updating](https://steamcommunity.com/app/221410/discussions/0/1734336452575505938/#c1734336452587441540), but [some works fine](https://steamcommunity.com/app/221410/discussions/0/1734336452575505938/#c2727382174636713310).

I did what you said and now [the game starts ok](https://www.youtube.com/watch?v=KXzElaH71a8) with the `Quake` option. `GLQuake` doesn't work (like you said it's a Win problem too) but it's a shame u_u haha

**Comment by aeikum** [comment]
Hello. With Proton 3.16-7, you can now use the `PROTON_OLD_GL_STRING` runtime option instead of the Mesa- or Nvidia-specific environment variables.

**Comment by jefferyrlc** [comment]
This game is crashing on launch it just spits out read CRASHLOG.txt for more inf
[CRASHLOG.TXT](https://github.com/ValveSoftware/Proton/files/8359695/CRASHLOG.TXT)
o
Arch Linux
Kernel 5.17.0-249-tkg-pds
Plasma 5.24.3 Wayland
AMD 5700XT
Mesa 21.3.7-2
